apps/settings/js/modules/storage/default_media_volume.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * DefaultMediaVolume:
+ *   - DefaultMediaVolume is an Observable that wraps the platform
+ *     DeviceStorage object.
+ *   - DefaultMediaVolume is a singleton that you can easily use it to fetch
+ *     some shared data across different panels.
+ *   - It has some observable properties: currentVolume.
+ * DefaultMediaVolume only update state and does not involve in any UI logic.
+ *
+ * @module DefaultMediaVolume
+ */
+define(function(require) {
+  'use strict';
+
+  var DeviceStorages = require('modules/storage/device_storages');
+  var Observable = require('modules/mvvm/observable');
```

Comment:
For example, when you define a read only observable property `currentVolume`, there will only be a getter exposed. Inside the object you can change the value of `_currentVolume`, then `currentVolume` changes along with `_currentVolume`.